ycmd/completers/rust/rust_completer.py

```diff
@@ -82,7 +74,7 @@ def _GetResponse( self, handler, request_data = {} ):
     were found.
     """
     self._logger.info('RustCompleter._GetResponse')
-    target = urlparse.urljoin( self._racerd_host, handler )
+    target = 'http://localhost:' + str(self._racerd_port) + str(handler)
```

Comment:
@vheon I removed self._racerd_host completely, it was generated from output of racerd command. It's using http://localhost prefix and port number from utils.GetUnusedLocalhostPort() now.